vllm/worker/habana_model_runner.py

```diff
@@ -1604,10 +1615,7 @@ def execute_model(
         if multi_modal_input is not None:
             execute_model_kwargs.update(multi_modal_input)
         if htorch.utils.internal.is_lazy():
-            execute_model_kwargs.update({
-                "bypass_hpu_graphs": not use_graphs,
-                "warmup_mode": warmup_mode
-            })
+            execute_model_kwargs.update({"bypass_hpu_graphs": not use_graphs})
```

Comment:
why is warmup_mode kwarg being removed here?

Reply:
It was also removed in original PR - should we leave that?

Reply:
Reverted